apps/nextjs-website/src/editorialComponents/Newsroom/Newsroom.tsx

```diff
@@ -117,7 +119,7 @@ const Item = (props: INewsroomItem) => {
           <LinkButton
             disabled={!!comingSoonLabel}
             href={href.link}
-            label={href.label}
+            label={t(href.label)}
```

Comment:
Because the label for href in input could be a translation string as happen in this file `apps/nextjs-website/src/app/[productSlug]/tutorials/page.tsx`  .